cfme/common/vm.py

```diff
@@ -183,7 +183,7 @@ def is_retired(self):
         view = navigate_to(self, "Details", use_resetter=False)
         if view.entities.summary('Lifecycle').get_text_of('Retirement Date').lower() != 'never':
             try:
-                status = view.entities.summary('Lifecycle').get_text_of('Retirement state').lower()
```

Comment:
Needs to be version picked, its lowercase 'state' in 59z

Reply:
@mshriver Right!